### DISCUSSION TITLE
Thoughts on documenting a release process/schedule

### DISCUSSION THREAD

**Discussion by jchorl** [nextflow-io/nextflow#1945] [Q&A] [answered]
Howdy!

It's been a few months, some good, some bad, but all filled with great nextflow progress!

I couldn't find a documented release schedule or process. Is this documented anywhere?

Even if it's just "I release when I have time" - might be worth adding to the README. There's some unreleased features that would be helpful to have.

I do know about the `-edge` releases. It's also not clear _how_ edge-y those are. Any guidance would be appreciated. If there's already words on this, feel free to point me at docs.

**Comment by pditommaso** [comment, marked as answer]
Hey there, we do edge release every month and bi-annual stable release in April and October. 

Edge releases contain the latest changes, it's expected to be functional but any new feature and improvement always require some time to be fully tested and we declared stable. Like for example https://github.com/nextflow-io/nextflow/pull/1862 😉 

Public roadmap: we don't have yet, however as the team expand as we are planning in the following weeks/months, we should be able to have a public roadmap
